tavern/tomes/file_list/metadata.yml

```diff
@@ -1,5 +1,5 @@
 name: List files
-description: List the files and directories found at the path
+description: List the files and directories found at the path. Supports basic glob functionality. Does not glob more than one level
 author: hulto
 support_model: FIRST_PARTY
 tactic: RECON
```

Comment:
Could you add an example of globbing to the placeholder.
Just `/etc/open*` probably.